src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Mashine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer почав працювати.");
+    }
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer закінчив працювати.");

```

Comment:
The messages in the `doWork` and `stopWork` methods should be in English. Consider changing them to something like "Bulldozer started working." and "Bulldozer stopped working." to meet the checklist requirement of using English and making messages informative.